### PR TITLE
changed | Quick commands -> Quick Commands

### PR DESCRIPTION
**Pull request by 4nshuman**
Resolves https://github.com/brave/brave-browser/issues/36845

Description:
"Quick commands" character case issue. Changed "Quick commands" to "Quick Commands".
This change reflects in both the hamburger menu as well as the "View" menu on title bar.

Snapshot references :
**_BEFORE -_** 
image
image

**_AFTER -_**
image
image

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
1. Start up Brave browser
2. Look at hamburger menu --> see "Quick commands" changed to "Quick Commands" 
3. Look at app menu (macOS) --> see "Quick commands" changed to "Quick Commands"

### CODE REVIEW

**Comment by 4nshuman** [comment]
Hello @bsclifton , @rebron,
Could you please review this PR.
Thanks.

**Comment by 4nshuman** [comment]
Hey @bsclifton , @rebron,
I'd love to hear your reviews on this PR.
Thanks.

File: app/brave_settings_strings.grdp [hunk 1]
```diff
@@ -138,7 +138,7 @@
     Bookmarks
   </message>
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_USE_AUTOCOMPLETE_COMMANDER" desc="The label for settings checkbox controlling whether or not commander suggestions show up in autocomplete">
-    Quick commands
+    Quick Commands
```
Comment (mkarolin) [inline app/brave_settings_strings.grdp:141]: This string is used in the Settings > Appearance > Show autocomplete suggestions in address bar. We have a bit of inconsistency there with capitalization, but I think `commands` (along with other entries) should not be capitalized there.

File: components/resources/commands.grdp [hunk 2]
```diff
@@ -11,7 +11,7 @@
     ChromeOS) prefer to not add the command.
   -->
   <message name="IDS_IDC_COMMANDER">
-    Quick commands
+    Quick Commands
```
Comment (mkarolin) [inline components/resources/commands.grdp:14]: This one is a bit trickier: we do want to capitalize it on MacOS, but not on other platforms. I think we may want to move it down in the file into `<if expr="use_titlecase">` section with capitalization and duplicate in `<if expr="not use_titlecase">` section without. cc: @fallaciousreasoning

Comment (fallaciousreasoning) [reply components/resources/commands.grdp:14]: Yup, I think thats a good call :smile:

File: app/brave_settings_strings.grdp (hunk 1 above)
Comment (4nshuman) [reply app/brave_settings_strings.grdp]: Hi @mkarolin, added the change, please take a look.

File: components/resources/commands.grdp (hunk 2 above)
Comment (4nshuman) [reply components/resources/commands.grdp]: Hello @mkarolin, @fallaciousreasoning,
I've added the conditional cases check. Please have a look.

**Comment by 4nshuman** [comment]
Thanks @fallaciousreasoning.
So will this PR not be merged ? Do we need a fresh PR ?

**Review by fallaciousreasoning — APPROVED** [review]
Nah, this PR will be merge - I just have to build it on another branch because we don't let PRs from external contributors run CI.

Basically, once https://github.com/brave/brave-core/pull/25700 is green we can merge this PR and #25700 will get closed. Its a pretty weird process :laughing:

**Comment by 4nshuman** [comment]
Got it, thanks @fallaciousreasoning

**Review by bsclifton — APPROVED** [review]
++

**Comment by bsclifton** [comment]
Should be safe since it's just strings - CI passed on https://github.com/brave/brave-core/pull/25700

Brave employees can see https://bravesoftware.slack.com/archives/CU5UXM3EC/p1727641977347059 for more info

**Comment by fallaciousreasoning** [comment]
Thanks for the contribution @4nshuman - sorry it took so long to get merged

**Comment by 4nshuman** [comment]
Thanks @fallaciousreasoning, @bsclifton.
The best part of this whole GFI was the local setup. It took some time to wrap my head around it, but a very rewarding experience.

I'll soon take up some more issues to contribute on !!

**Pull request merged**